pypeit/data/arc_lines/lists/ArI_lines.dat

```diff
@@ -11,7 +11,6 @@
 # Creation Date: 2016-Dec-21
 | ion |       wave | NIST | Instr | amplitude |                     Source |
 | ArI |  3950.0970 |    1 |   128 |     10000 |                       NIST |
-| ArI |  4045.5610 |    1 |   128 |     10000 |                       NIST |
```

Comment:
I presume this line was behaving poorly?

Reply:
Yes, it's very close to another line (HgI 4047.71). But I'll double check with other instruments that it's fine to remove.